### PR TITLE
Add a migration to have markets.yml migrated to the database

### PR DESCRIPTION
**Pull request by sramsden**
Add a migration to have config/markets.yml migrated to the database markets table rather than having to copy/edit yml then run seed task .. but leave the seed task, however ensure it doesn't fail if the seed yml does not exist.
The migration will simply do nothing if there is no config/markets.yml file available to migrate and it will not overwrite any existing markets in the database either (matching via market id)

### CODE REVIEW

**Comment by yivo** [comment]
@yssavchuk Could you please take a look at code logic, code style etc.

File: spec/migrations/migrate_markets_spec.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                position: '1',
+                bid_fee: '0.0015',
+                bid_precision: '2',
+                ask_fee: '0.0015',
+                ask_precision: '4'
+              }
+            )
+            subject
+          end
+
```
Comment (ysv) [inline spec/migrations/migrate_markets_spec.rb]: @sramsden remove blank lines please

Comment (yivo) [reply spec/migrations/migrate_markets_spec.rb]: @yssavchuk Anything else related to code?

Comment (sramsden) [reply spec/migrations/migrate_markets_spec.rb]: Blank line make it easier to determine which block they are ending.
I can remove the blank like but add a comment after the `end` to specify what it is ending.
Which would you prefer?

**Comment by sramsden** [comment]
I have just noticed that all my trades still reference `market_id` from the old yml market `code` .. eg, `101` for `btcusd`
I will therefore make a change to update the trades, but just wanted to ask how you guys were managing this without my data migration?   Your existing https://github.com/rubykube/peatio/blob/master/docs/releases/1.6.0.md make no mention of updating the trades data.
Am I missing something?

**Comment by yivo** [comment]
@sramsden We used this: https://github.com/rubykube/peatio/blob/master/db/migrate/20180329154130_update_deprecated_currency_ids.rb

**Comment by sramsden** [comment]
@yivo okay.  I assume it didn't work for me  because I must have migrated before renaming to old yml etc.
It's getting confusing so I am going to close the PR and just keep my code on my fork.  You guys don't really need it.

**Pull request closed without merging**